<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="m_PlayBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>16</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton{
     background-color: red;
 }
 QPushButton:pressed {
     background-color: green;     
 }
 QPushButton:hover :!pressed{
     background-color: red;
 }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_SettingsBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>16</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton{
     background-color: red;
 }
 QPushButton:pressed {
     background-color: green;     
 }
 QPushButton:hover :!pressed{
     background-color: red;
 }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="m_QuitBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>16</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton{
     background-color: red;
 }
 QPushButton:pressed {
     background-color: green;     
 }
 QPushButton:hover:!pressed {
     background-color: yellow;
 }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
